src/app/Components/Artist/ArtistAbout/ArtistAboutShows.tsx

```diff
@@ -12,94 +10,43 @@ interface Props {
 }
 
 const ArtistAboutShows: React.FC<Props> = ({ artist }) => {
-  const currentShows = extractNodes(artist.currentShows)
-  const upcomingShows = extractNodes(artist.upcomingShows)
-  const currentAndUpcomingShows = [...currentShows, ...upcomingShows]
+  const shows = extractNodes(artist?.showsConnection)
+  const { width } = useScreenDimensions()
+  const { space } = useTheme()
 
-  const pastShows = extractNodes(artist.pastShows)
-  const screenWidth = useScreenDimensions().width
-
-  // We show the current and upcoming shows. If no current/upcoming, we show the 3 past shows
-  // See https://artsyproduct.atlassian.net/browse/CX-743 for business rules
-  const shownShows = currentAndUpcomingShows.length > 0 ? currentAndUpcomingShows : pastShows
-
-  const userHasShows = currentAndUpcomingShows.length + pastShows.length
-
-  if (userHasShows) {
-    return (
-      <Flex>
-        <Text variant="sm-display" mb={1}>
-          Shows featuring {artist.name}
-        </Text>
-        <FlatList
-          data={shownShows}
-          renderItem={({ item }) => (
-            <ArtistShowFragmentContainer
-              show={item}
-              styles={{
-                container: {
-                  width: 335,
-                  marginRight: 15,
-                },
-                image: {
-                  width: 335,
-                  height: 220,
-                  marginBottom: 10,
-                },
-              }}
-            />
-          )}
-          ItemSeparatorComponent={() => <Spacer x={1} />}
-          keyExtractor={(show) => show.id}
-          showsHorizontalScrollIndicator={false}
-          horizontal
-          style={{ left: -20, width: screenWidth }}
-          contentContainerStyle={{ paddingBottom: 15, paddingLeft: 20 }}
-        />
-        {!!pastShows.length && (
-          <Button
-            variant="fillGray"
-            onPress={() => navigate(`/artist/${artist?.slug!}/shows`)}
-            size="small"
-            block
-          >
-            See all past shows
-          </Button>
-        )}
-      </Flex>
-    )
+  if (shows.length === 0 || !artist) {
+    return null
   }
 
-  // If the user has no past/current/upcoming shows
-  return null
+  return (
+    <Flex>
+      <Text variant="sm-display" pb={4}>
+        Shows Featuring {artist.name}
+      </Text>
+      <FlatList
+        data={shows}
+        renderItem={({ item }) => <ArtistAboutShow show={item} />}
+        ItemSeparatorComponent={() => <Spacer x={1} />}
```

Comment:
question(non-blocking): Looking into [figma](https://www.figma.com/file/h55uSy8xjTBSQHBhY8mLUw/Artist-Page-2023?type=design&node-id=1651-29077&mode=design&t=ubb6q2QWcJ5LAn7L-4) and I think this should be 2 to match also the other rails?

```suggestion
        ItemSeparatorComponent={() => <Spacer x={2} />}
```

Reply:
nice catch!